dotCMS/src/main/java/com/dotcms/aspects/interceptors/CloseDBMethodInterceptor.java

```diff
@@ -0,0 +1,38 @@
+package com.dotcms.aspects.interceptors;
+
+import static com.dotcms.util.AnnotationUtils.getMethodAnnotation;
+
+import com.dotcms.aspects.DelegateMethodInvocation;
+import com.dotcms.aspects.MethodInterceptor;
+import com.dotcms.business.CloseDB;
+
+
+import com.dotmarketing.db.DbConnectionFactory;
+
+/**
+ * Method handler for the {@link CloseDBIfOpened} annotation aspect
+ * @author jsanca
+ */
+public class CloseDBMethodInterceptor implements MethodInterceptor<Object> {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)